jscomp/ml/lexer.mll

```diff
@@ -254,7 +254,7 @@ let report_error ppf = function
   | Unterminated_string_in_comment (_, loc) ->
       fprintf ppf "This comment contains an unterminated string literal@.\
                    %aString literal begins here"
-              Location.print_error loc
+              (Location.print_error "") loc
```

Comment:
Why this change? Has the type of `print_error` changed?

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6199#discussion_r1179521031